installer.sh

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env bash
+
+## Some environment setup
+set -e
+export DEBIAN_FRONTEND=noninteractive
+
+##
+
+## Handle Arguments
+
+if [[ ! -n $1 ]]; then
+  echo "No arguments provided, installing with"
+  echo "default configuration values."
+fi
+
+while [ $# -gt 0 ]; do
+  case "$1" in
+    --travis_worker_version=*)
+      TRAVIS_WORKER_VERSION="${1#*=}"
+      ;;
+    --docker_version=*)
+      DOCKER_VERSION="${1#*=}"
+      ;;
+    --aws=*)
+      AWS="${1#*=}"
```

Comment:
Looks like we're not using this anymore to setup the docker volume in `/mnt`? Do we no longer need to set the docker dir to that with the newer instance types? Before we'd run out of space.

Reply:
Ah, good catch here! Right now I'm not setting it, therefore I'd get rid of it. What I did on my test machines is to configure the main volume to have >= 40GB to accommodate the build images, which should be fine? :)